src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/AlleleFilteringHC.java

```diff
@@ -57,14 +60,14 @@ int getAlleleLikelihoodVsInverse(final AlleleLikelihoods<GATKRead, Allele> allel
 
         final GenotypingLikelihoods<Allele> genotypingLikelihoods = genotypesModel.calculateLikelihoods(alleleList,
                 genotypingData, null, 0, null);
-        AFCalculationResult af = afCalc.fastCalculateDiploidBasedOnGLs(genotypingLikelihoods, genotypingEngine.getPloidyModel().totalPloidy());
-        final double log10Confidence = af.log10ProbOnlyRefAlleleExists();
-        final double phredScaledConfidence = (10.0 * log10Confidence) + 0.0;
 
-        final int[] asPL = genotypingLikelihoods.sampleLikelihoods(0).getAsPLs();
-
-        logger.debug(() -> String.format("GAL:: %s: %d %d %d", allele.toString(), asPL[0], asPL[1], asPL[2]));
-        return Math.min(asPL[1]-asPL[0], asPL[2]-asPL[0]);
+        List<Integer> perSamplePLs = new ArrayList<>();
+        for (int i = 0; i < genotypingLikelihoods.numberOfSamples(); i++) {
+            final int[] pls = genotypingLikelihoods.sampleLikelihoods(i).getAsPLs();
+            perSamplePLs.add(Math.min(pls[1] - pls[0], pls[2] - pls[0]));
+            logger.debug(() -> String.format("GAL:: %s: %d %d %d", allele.toString(), pls[0], pls[1], pls[2]));
```

Comment:
Now that you have multiple samples here you might want to add the sample name to the debug logger